### ISSUE TITLE
Large spacing in 'cannot create indexes' dialog

### ISSUE THREAD

**Issue by chrisjlocke** [sqlitebrowser/sqlitebrowser#771]
### Details for the issue

Added a field to my database, and DB4S struggled to put back some indexes.  Bit weird, as there was no reason why it shouldn't have been able to do it - I was adding a field, not deleting any.  Aah well, no biggie.  However, the dialog was a bit 'caca' with a huge gap.

![image](https://cloud.githubusercontent.com/assets/7872218/18378407/bc2ed4dc-7664-11e6-84f8-89304792c262.png)

This was with the nightly, so not sure if its in 3.9 ... will try again and have a look...

#### I'm opening this issue because:

- [ ] DB4S is crashing
- [x] DB4S has a bug
- [ ] DB4S needs a feature
- [ ] DB4S has another problem

#### I'm using DB4S on:

- [x] Windows
- [ ] Linux: ( _distro:_ ___ )
- [ ] Mac OS
- [ ] Other: ___

#### I'm using DB4S version:

- [ ] 3.9.0
- [ ] 3.8.0
- [x] Other: 3.9.99

#### I have also:

- [x] Tried out the nightly version: https://github.com/sqlitebrowser/sqlitebrowser#windows
- [ ] Searched for an existing similar issue: https://github.com/sqlitebrowser/sqlitebrowser/issues?utf8=%E2%9C%93&q=is%3Aissue%20

**Comment by chrisjlocke** [comment]
I assume there is no log as to why recreating these indexes failed?  Running the statements in the above dialog worked flawlessly.  Nothing appeared in the SQL log...

**Comment by justinclift** [comment]
Interesting.  This kind of thing suggests that writing to a log somewhere is probably a good idea.

If you can reproduce the problem, try starting DB4S from the command line (eg from DOS window, Powershell, etc).  Sometimes debug messages turn up there, so it's probably a useful thing to check, just in case. :smile:

**Comment by MKleusberg** [comment]
I *think* this bug should be fixed with the latest commit. DB4S now removes any whitespace from the SQL commands shown in the message box and this sort of whitespace is the only reason I can imagine the message box looking like this.
Also we now put a semicolon at the end of each SQL command. Before, those two commands weren't separated by one and that might have been the cause of the message box to begin with.

So in short: the whitespace should be gone. Also you hopefully won't be able to confirm this so easily because that error shouldn't appear anymore :wink:

**Comment by chrisjlocke** [comment]
Ex cellent!  The 'we can't recreate the indexes' was popping up a lot, and guess (in hindsight) I can see why now, as those tables would have had more than one index...

**Comment by MKleusberg** [comment]
I had exactly the same thought ^^ It all makes sense now.
Do you have the time to do one little test or two with the next nightly build, so we can be sure it's safe to close this issue?

**Comment by chrisjlocke** [comment]
Will gladly test tomorrows nightly. 😀

**Comment by revolter** [comment]
@chrisjlocke Did you manage to test this? 😄

**Comment by chrisjlocke** [comment]
I've tested the nightlies, and can't reproduce the error, so thats a good thing!  Consider it closed, and I'll yell if it comes up again! 😉
